Assets/Scripts/Room/Chat/ChatMessage.cs

```diff
@@ -0,0 +1,122 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Localization.Settings;
+using System.Collections;
+using Assets.Scripts.UI.Input;
+
+namespace Assets.Scripts.Room.Chat
+{
+    public class ChatMessage : MonoBehaviour
```

Comment:
Add a comment that describes the class, according to https://github.com/AoTTG-2/AoTTG-2/wiki/Code-&-Style-Conventions#comments

Reply:
On it 👍